libs/wingc/src/jsify.rs

```diff
@@ -1215,7 +1225,11 @@ impl<'a> JSifier<'a> {
 
 		// if this class has a "handle" method, we are going to turn it into a callable function
 		// so that instances of this class can also be called like regular functions
-		if inflight_methods.iter().find(|(name, _)| name.name == HANDLE_METHOD_NAME).is_some() {
+		if inflight_methods
+			.iter()
+			.find(|(name, _)| name.name == HANDLE_METHOD_NAME)
```

Comment:
Nit: Use `any` instead?

Reply:
this was cleaned up in the inflight classes PR so not relevant anymore